getting_started/access_your_services.adoc

```diff
@@ -180,17 +223,94 @@ Endpoints:                <none>
+to xref:../architecture/core_concepts/pods_and_services.adoc#architecture-core-concepts-pods-and-services[pods and services]. A link:https://kubernetes.io/docs/concepts/services-networking/#loadbalancer[LoadBalancer]
+service creates an AWS Elastic Load Balancer (ELB) for your {product-title} cluster, enabling direct TCP access to applications exposed by your LoadBalancer service.
+
+[NOTE]
+====
+LoadBalancer services require an additional purchase. Contact your sales team if you are interested in using LoadBalancer services for your {product-title} cluster.
+====
+
+=== Check your LoadBalancer Quota
+
+By purchasing LoadBalancer services, you are provided with a quota of LoadBalancers available for your {product-title} cluster.
+
+----
+$ oc describe clusterresourcequota service-loadbalancers
+Name:       service-loadbalancers
+Labels:     <none>
+Annotations:    <none>
+Resource        Used    Hard
+--------        ----    ----
+services.loadbalancers  0   4
+----
+
+=== Expose TCP Service
+
+You can expose your applications over an external LoadBalancer service, enabling access over the public Internet.
+
+----
+$ oc expose dc httpd-example --type=LoadBalancer --name=lb-service
+service/lb-service exposed
+----
+
+=== Create an Internal-Only TCP Service
+
+You can alternatively expose your applications internally only, enabling access only through AWS VPC Peering or a VPN connection.
+
+----
+$ oc expose dc httpd-example --type=LoadBalancer --name=internal-lb --dry-run -o yaml | awk '1;/metadata:/{ print "  annotations:\n    service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0" }' | oc create -f -
+service/internal-lb exposed
+----
+
+=== Use your TCP Service
+
+Once your LoadBalancer service is created, you can access your service by using the URL provided to you by {product-title}.
+The `LoadBalancer Ingress` value is a URL unique to your service that remains static as long as the service is not deleted.
+If you prefer to use a custom domain, you can create a CNAME DNS record for this URL.
+
+----
+$ oc describe svc lb-service
+Name:                     lb-service
+Namespace:                default
+Labels:                   app=httpd-example
+Annotations:              <none>
+Selector:                 name=httpd-example
+Type:                     LoadBalancer
+IP:                       10.120.182.252
+LoadBalancer Ingress:     a5387ba36201e11e9ba901267fd7abb0-1406434805.us-east-1.elb.amazonaws.com
+Port:                     <unset>  8080/TCP
+TargetPort:               8080/TCP
+NodePort:                 <unset>  31409/TCP
+Endpoints:                <none>
+Session Affinity:         None
+External Traffic Policy:  Cluster
+----
+endif::[]
 
 [[access-your-cluster-status-updates]]
 == Receive status updates
 
+// TODO Need to check to see if Microsoft is providing this
```

Comment:
Overall service status/health information is available at https://azure.microsoft.com/en-us/status/